mewarpx/medocker/docker_ubuntu2004/Dockerfile

```diff
@@ -1,30 +1,30 @@
 ARG BUILDPLATFORM
 
-ARG WARPXCOMPUTE=OMP
+ARG WARPXCOMPUTE
 
 FROM 167833485543.dkr.ecr.us-west-2.amazonaws.com/simteam/ubuntu-mirror:focal-${BUILDPLATFORM}
 
-ENV WARPXCOMPUTE=${WARPXCOMPUTE}
+ENV WARPXCOMPUTE=${WARPXCOMPUTE:-OMP}
 
 ARG MPI_IMPL=ompi
 
 # The '--login' is the way to get mpich to load happily, which in
 # turn is necessary for a successful warp pinstall (and any mpi command
```

Comment:
Outdated comment.

Reply:
I updated the comment and turned on auto-merge

Reply:
Thanks for taking care of that!